### PR TITLE
Add template Schema api views

### PR DESCRIPTION
**Pull request by CloudNiner**

### CODE REVIEW

**Comment by CloudNiner** [comment]
I left the flask-restful commits in just in case we need to use it instead of restless due to the geo serialization/filtering issue.

File: app/ashlar/api/__init__.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+from flask.ext.restless import APIManager
+
+from ashlar import app, db
+from ashlar.ashlar.exceptions import SchemaException
+from ashlar.ashlar.models import Record, RecordSchema, ItemSchema
+
+
+api_manager = APIManager(app, session=db.session)
+
+## TODO: Figure out how to properly serialize the geo column
```
Comment (flibbertigibbet) [inline app/ashlar/api/__init__.py:13]: Why not use [`ST_AsGeoJSON`](http://geoalchemy-2.readthedocs.org/en/latest/spatial_functions.html)?

Comment (CloudNiner) [inline app/ashlar/api/__init__.py:13]: Same issue as AsText, see below.

**Comment by ddohler** [comment]
+1, hopefully we can get the Geometry issue figured out.

**Pull request merged**